src/main/resources/application.yml

```diff
@@ -1,17 +1,17 @@
 spring:
   servlet:
     multipart:
-      max-file-size: 2MB
-      max-request-size: 2MB
+      max-file-size: 10MB
+      max-request-size: 10MB
   mvc:
     static-path-pattern: /static/**
 
   datasource:
     platform: postgres
     driverClassName: org.postgresql.Driver
-    url: jdbc:postgresql://edu-db:5432/edumanagement
-    username: root
-    password: root
+    url: jdbc:postgresql://edumanagement.cp7ompjjcxv5.eu-north-1.rds.amazonaws.com/postgres
```

Comment:
Add a spring profile for AWS deployments, and change the variables in that profile, (e.g. application-ebs.yml)